app/views/teachers/RequestQuoteView.coffee

```diff
@@ -8,15 +8,16 @@ ConfirmModal = require 'views/editor/modal/ConfirmModal'
 algolia = require 'core/services/algolia'
 
 SIGNUP_REDIRECT = '/teachers'
-NCES_KEYS = ['id', 'name', 'district', 'district_id', 'district_schools', 'district_students', 'students', 'phone']
+DISTRICT_NCES_KEYS = ['district', 'district_id', 'district_schools', 'district_students', 'phone']
+SCHOOL_NCES_KEYS = DISTRICT_NCES_KEYS.concat(['id', 'name', 'students'])
 
 module.exports = class RequestQuoteView extends RootView
   id: 'request-quote-view'
   template: require 'templates/teachers/request-quote-view'
   logoutRedirectURL: null
 
   events:
-    'change #request-form': 'onChangeRequestForm'
+    'change #request-form': 'onChangeForm'
```

Comment:
There are two forms in this view, a request form and a signup form, so it would be better to keep this specifically named.